tests/bluetooth/tester/src/audio/btp_csis.c

```diff
@@ -132,6 +142,8 @@ uint8_t tester_init_csis(void)
 	};
 	int err = bt_csip_set_member_register(&register_params, &csis_svc_inst);
 
+	sirk_read_response = BT_CSIP_READ_SIRK_REQ_RSP_REJECT;
```

Comment:
Why are we defaulting to rejecting? 

IMO it probably makes more sense to default to `BT_CSIP_READ_SIRK_REQ_RSP_ACCEPT` as that's the most commonly used response type